extensions/markdown-language-features/preview-src/index.ts

```diff
@@ -146,23 +147,31 @@ async function copyImage(image: HTMLImageElement, retries = 5) {
 	}
 
 	try {
-		await navigator.clipboard.write([new ClipboardItem({
-			'image/png': new Promise((resolve) => {
+		const imageSrc = image.getAttribute('data-src') || 'src-not-found';
+
+		if (isURL(imageSrc)) {
+			await navigator.clipboard.writeText(imageSrc);
```

Comment:
Instead of writing text, maybe use apis to select the image and then use `execCommand('copy')`  tocopy it. It still won't write out image data but it will better match what browser do. Most of them write out `text/html` too for example